packages/merkle-distributor/src/merkleTree.js

```diff
@@ -0,0 +1,131 @@
+const web3 = require("web3");
+const ethereumjsUtil = require("ethereumjs-util");
+
+class MerkleTree {
+  constructor(elements) {
+    // Filter empty strings and hash elements
+    // NOTE do not double has leafs
```

Comment:
typo?

Reply:
Probably? @chrismaree wrote this

Reply:
nope, I did not. this script was copied directly from the Piedao implementation, found [here](https://github.com/pie-dao/vested-token-migration-app/blob/master/scripts/merkleTree.js). The one change I did make was changing this to work with web3 over Ethers.


Note that in the SDK version I'm building out on a separate PR this file is moved over to typescript and copies the uniswap implementation.

Reply:
Can we drop this file from this PR then? Or do we still need it?

Reply:
it's needed in the unit tests to generate the proofs.